First pass on new technical issue template (#16)
Medology/.github: .github/ISSUE_TEMPLATE/Standard-Technical-Issue.md

## Description

< Description of the problem. >

## Negative Impact:

> < what problems does, will, or can the problem described cause for Dev, QA, or DevOps. >

Reviewer: The English here is a little off

```suggestion
< How the described problem does, will, or can affect Dev, QA, or DevOps. >
```

## Proposed Solution:

> < Describe the propsed solution for the problem. >

Reviewer: Typo

```suggestion
< Describe the proposed solution for the problem. >
```
